arches_querysets/rest_framework/serializers.py

```diff
@@ -436,6 +437,7 @@ def create(self, validated_data):
             dummy_instance.sortorder = None
             dummy_instance.set_next_sort_order()
             validated_data["sortorder"] = dummy_instance.sortorder
+        validated_data["__request"] = self.context["request"]
```

Comment:
... but while here I noticed that the user wasn't getting attached. I'd love to only pass the user around, but we have some baggage in core re: tile functions that expect a request object.